src/workloads/query/TenMDocCollection_IntId.yml

```diff
@@ -1,109 +1,172 @@
+  Config:
+    LoadConfig:
+      Path: TenMDocCollection_Base.tmpl
+      Key: ActorTemplateConfig
+      Parameters:
+        MaxPhases: *MaxPhases
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {_id: {^RandomInt: {min: 0, max: 9999999}}}
+            Options:
+              Projection: {_id: 1}
+- TemplateName: IDFieldEq
+  Config:
+    LoadConfig:
+      Path: TenMDocCollection_Base.tmpl
+      Key: ActorTemplateConfig
+      Parameters:
+        MaxPhases: *MaxPhases
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {a: {$eq: {^RandomInt: {min: 10000000, max: 19999999}}}}
+- TemplateName: IDFieldIn
+  Config:
+    LoadConfig:
+      Path: TenMDocCollection_Base.tmpl
+      Key: ActorTemplateConfig
+      Parameters:
+        MaxPhases: *MaxPhases
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter: {a: {$in: {^Array: {
+              distinct: true,
+              number: {^Parameter: {Name: "Number", Default: 100000}},
+              of: {^RandomInt: {min: 10000000, max: 19999999}}}}}}
+- TemplateName: IDFieldOr
+  Config:
+    LoadConfig:
+      Path: TenMDocCollection_Base.tmpl
+      Key: ActorTemplateConfig
+      Parameters:
+        MaxPhases: *MaxPhases
+        Operations:
+        - OperationName: find
+          OperationCommand:
+            Filter:
+              $or:
+                ^Array:
+                  distinct: true
+                  number: 10
+                  of:
+                    $and:
+                    - b: {$gte: {^RandomInt: {min: 20000000, max: 29999999}}}
+                    - b: {$lt: {^RandomInt: {min: 20000000, max: 29999999}}}
 
+Actors:
+- LoadConfig:
+    Path: TenMDocCollection_Base.tmpl
+    Key: ClearCollectionActor
+    Parameters:
+      MaxPhases: *MaxPhases
+- LoadConfig:
+    Path: TenMDocCollection_Base.tmpl
+    Key: PrepopulateActor
+    Parameters:
+      Document:
+        # The multiplier is multiplied by the ActorID, and added to the value.
+        # Since this is the second actor, the ActorIDs increment for each thread starting at 2.
+        # e.g -2000000 + 2 * 1000000 = 0, -1999999 + 2 * 1000000 = 1, etc.
+        # Once complete, the ids in the collection will be the range 0-9999999.
+        _id: {^Inc: {start: -2000000, multiplier: 1000000}}
+        # [10M, 20M)
+        a: {^Inc: {start: -1000000, multiplier: 1000000}}
+        # [20M, 30M)
+        b: {^Inc: {start: 0, multiplier: 1000000}}
+      Indexes:
+        - keys: {a: 1}
+        - keys: {b: 1}
+        - keys: {a: -1, b: -1}
+      MaxPhases: *MaxPhases
+- LoadConfig:
+    Path: TenMDocCollection_Base.tmpl
+    Key: QuiesceActor
+    Parameters:
+      MaxPhases: *MaxPhases
 - ActorFromTemplate:
     TemplateName: IDHACK
     TemplateParameters:
-      Name: IDHack32Threads
-      Threads: 32
+      Name: IDHACK032Threads
```

Comment:
can you please avoid changing these names as part of this commit? Can we throw a separate cleanup ticket in to do it? I think it is a good change, but it will make all our graphs break and I think we will stop caring about our graphs soon enough that we can/should just defer this.

Reply:
Don't think we'll need a cleanup ticket for this, since there aren't as many 32t tests now.